src/client.ts

```diff
@@ -403,4 +399,87 @@ export default class Client implements ClientInterface {
       return false;
     }
   }
+
+  private async gemsAreInstalled(customGemfilePath: string): Promise<boolean> {
+    try {
+      await asyncExec(`BUNDLE_GEMFILE=${customGemfilePath} bundle check`, {
+        cwd: this.workingFolder,
+      });
+      return true;
+    } catch {
```

Comment:
Worth logging the exception here?

Reply:
When gems are not installed, running bundle check will return status 1 and fail, which leads us to this branch. There's not much to learn from logging here, this just indicates gems aren't installed.